### PR TITLE
Refine nudge and Select My Business page

### PR DESCRIPTION
**Pull request by Tug**
This PR refines the nudge and my business page to fit the last version of the design.

Checklist:
- [x] Fix height of nudge that changes depending on whether image has loaded
- [x] Fix `I'm Already Listed` button that doesn't close the nudge
- [x] Fix nudge shown very quickly while page loads even if it has been dismissed
- [x] Fix the size of the phone to display smaller (Nudge)
- [x] Fix the width of the type on the second screen to wrap sooner. See comment below...
- [x] Fix the spacing between headings and descriptions
- [x] Fix the "Optimize Your SEO" CTA so that it doesn't open in a new tab.

Not sure about this one but it also compresses the phone screenshot which was a bit heavy (277KB) by removing mostly invisible parts and using an 8bit color palette. It weights 150KB now).

### Testing Instructions
- Test that the nudge looks like the mockup and that the copy is right
- Test that the buttons work properly as described in the design post (i3)

### Reviews
- [ ] Code
- [ ] Product

### CODE REVIEW

File: client/my-sites/google-my-business/select-business-type/style.scss [hunk 1]
```diff
@@ -41,25 +44,33 @@
...
 		font-weight: 400;
 	}
 
 	img {
-		max-width: 300px;
+		max-width: 400px;
+		width: 150%;
 		align-self: center;
+
+		@include breakpoint( "<1040px" ) {
```
Comment (scruffian) [inline client/my-sites/google-my-business/select-business-type/style.scss]: We should try to use > breakpoints when possible to reduce the rendering load for mobiles

**Comment by scruffian** [comment]
I just had a chat with @cburton4 about this and we have three pieces of feedback:

1. We should remove the "I'm Already Listed" button. Users wont know what it means, and it just creates confusion

2. At large screen sizes, this image looks too big, and it becomes difficult to see what is being shown on the phone
screen shot 2018-02-27 at 21 47 10

3. At small screen sizes, this the image on the phone is missing the photo section, so it looks weird:
screen shot 2018-02-27 at 21 47 00

**Comment by cburton4** [comment]
In regards to the `I'm Already Listed` button, we had a conversation about customers likely not understanding what this means and could cause confusion. Also knowing that this button adds more development time makes it even less important.

**Comment by yurynix** [comment]
Looks good, LGTM 🚢

**Pull request merged**